org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/outline/OutlinePanelView.java

```diff
@@ -323,9 +323,9 @@ public void valueChanged(TreeSelectionEvent e)
       @Override
       public void actionPerformed(ActionEvent e)
       {
-        if(_cutContents!=null) {

```

Comment:
@ayeshare - it's not good practice to leave the line commented out like this - the future Ian (or even the current Ian) won't understand what's happening.

If we don't need the code we should delete it.  Or, add a comment to explain why it's commented out, please.